src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationVault.adoc

```diff
@@ -0,0 +1,29 @@
+Since 1.1, Micronaut features a native https://www.vaultproject.io/[HashiCorp Vault] for those who have not switched to a dedicated more complete solution like Consul.
+
+To enable support for Vault Configuration simply add the following configuration to your `bootstrap.yml` file:
+
+.Integrating with HashiCorp Vault
+[source,yaml]
```

Comment:
Instead of describing each key manually, include the configuration reference table

Reply:
Got it. will do that.